src/pages/stakeholders/index.md

```diff
@@ -0,0 +1,85 @@
+
+Shortly after that, I created [journey maps](https://blog-nrrd.doi.gov/journey-mapping/) to understand how the whole process works. I created the map after interviewing ONRR employees as potential users of a future lease level dataset for the NRRD website. This helped me to understand how the process fits together to some extent, but not at the level needed for ONRR.gov.
+
+When [we started working on ONRR.gov](https://blog-nrrd.doi.gov/adding-a-product/), we interviewed industry and internal users about how they use the site. This gave us an understanding of the tasks users complete with the site and some of the pain points with the process. It also allowed us to solidify our [product statement](https://github.com/ONRR/onrr.gov-site/wiki/Product-framing#problem-statement), [vision](https://github.com/ONRR/onrr.gov-site/wiki/Product-framing#our-vision), and [user types](https://github.com/ONRR/onrr.gov-site/wiki/Product-framing#user-types-and-scenarios).
+
+### ONRR.gov problem statement
+
+> Companies pay to produce natural resources on federal lands, Indian lands, and the Outer Continental Shelf. They need to access timely and accurate information to meet complex regulatory requirements. These requirements include reporting production and paying the required royalties and other revenues.  The Office of Natural Resources Revenue collects, verifies, and disburses those revenues.  
+
+> Native Americans and the public need to understand their revenues and ensure we meet our trust responsibilities. ONRR should provide access to resources and clear communication to help this understanding.
+
+### Vision
+
+> We communicate the role of the Office of Natural Resources Revenue. We deliver trusted and easy to use information and customer service. This enables companies who lease federal and Indian lands to accurately report production and pay revenue due.​
+
+### User types
+
+New production and revenue reporter​s use the site to request access to reporting systems. They also learn about production and royalty reporting and payment requirements.
+
+Existing reporters and payors use the site monthly to access ONRR’s reporting portal. They have to submit reports for their oil, gas, geothermal, or solid minerals production and revenue. These users look on ONRR.gov for handbooks, pricing information, and lease numbers or codes to put on their reports. They also use information about submitting payment of the amount they owe ONRR. Some may look for information about compliance if they get a letter about enforcement.
+
+ONRR customer service rep​s are internal users who use the site to provide information to reporters and payors.
+
+Inquisitor​s are people like job seekers or journalists who use the site to learn about ONRR.
+
+Resource owners are tribal representatives or Individual Indian Mineral Owners (IIMOs) who own mineral rights. They use the site to get information about ONRR’s trust responsibilities and their leases.
```

Comment:
Resource owners are tribal representatives or Individual Indian Mineral Owners (IIMO's) who own mineral rights. They use the site to get information about ONRR’s trust responsibilities and their leases.

Reply:
IIMOs does not need an apostrophe in this context.  In this context they are plural and not possessive
.

Reply:
Acronyms and Initialisms use apostrophes for both plural and possessive cases

Reply:
This is from the GPO Style Manual:  
8.11.    While an apostrophe is used to indicate possession and contractions,
it is not generally necessary to use an apostrophe simply to
show the plural form of most acronyms, initialisms, or abbreviations,
except where clarity and sense demand such inclusion.


It may be useful for the team to work with Matt Williams to get a list of common acronyms and how they appear in different context.  This is something that would be useful ONRR-wide.

Reply:
I strongly disagree with this guidance, I like clear binary standards.  But that's just me!